src/middlewares/auth.middleware.ts

```diff
@@ -3,11 +3,16 @@ import { verifyJWT } from '@services/jwt.service';
 
 export default class {
   static async checker(action: Action): Promise<boolean> {
-    const token = action.request.headers['authorization'];
+    let token = action.request.headers['authorization'];
     if (!token) {
       return false;
     }
 
+    if (token.startsWith('Bearer ')) {
```

Comment:
is this required to do swagger login?

Reply:
Basically, the standard way is to send the authentication scheme in the header (Basic, Bearer, Digest, etc). This new condition supports both approaches, with and without the scheme.